internal/server/data/organization.go

```diff
@@ -158,3 +158,28 @@ func UpdateOrganization(tx WriteTxn, org *models.Organization) error {
 func CountOrganizations(tx ReadTxn) (int64, error) {
 	return countRows(tx, organizationsTable{})
 }
+
+type ForgottenDomain struct {
+	OrganizationName   string
+	OrganizationDomain string
+	// LastSeenAt is the last time the user logged into this organization.
+	LastSeenAt time.Time
```

Comment:
This type was moved from `models`. 

> This type is not really a model. I doesn't have CRUD operations, and it
doesn't act as a representation of a database table.

> Types like this (see data.DestinationsCount for another example) are
nothing more than return values for a single data function. Moving it
to the data package makes this more clear. It also makes the code much
easier to trace because you don't have to jump into a completely
separate file and package when looking up the type.

`LastSeenAt` was also changed from a `string` to a `time.Time` which made testing this with strict assertions much easier. The formatting of the time moved to the template, which is covered by a test.